libraries/ms-helper/inc/exported_enums.h

```diff
@@ -25,6 +25,13 @@ typedef enum {
   NUM_EE_CHARGER_CONN_STATES,
 } EEChargerConnState;
 
+// General relay state
```

Comment:
Pretty sure I already added this in my relay refactoring... Merge?

Reply:
Yeah, I just moved it up in the file.